src/overlays/actors/ovl_En_Floormas/z_en_floormas.h

```diff
@@ -4,9 +4,24 @@
 #include <ultra64.h>
 #include <global.h>
 
-typedef struct {
+#define SPAWN_INVISIBLE 0x8000
+#define SPAWN_SMALL 0x10
+
+// Merge params
+#define MERGE_MASTER 0x40
+#define MERGE_SLAVE 0x20
+
+typedef struct EnFloormas{
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1C8];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ void (*actionFunc)(struct EnFloormas* this, GlobalContext* globalCtx);
```

Comment:
we've decided we are going to go with typedefs for these. there are some examples in discord, if you wanted to start matching that pattern now